### DISCUSSION TITLE
Any OTA update on non-A/B slot device?

### DISCUSSION THREAD

**Discussion by Pr0pHesyer** [bmax121/APatch#498] [Q&A] [answered]
I use Mi6 still,it doesn't support A/B slot.I could not install to inactive slot after OTA.

**Comment by marat2509** [comment]
Since you can do something with a non-existent second slot (since it is not supported). Does your device have boot_a and boot_b partitions, or just boot partition?

**Reply by Pr0pHesyer** [reply]
Just boot partition.

**Reply by marat2509** [reply, marked as answer]
So you have non-A/B device, and it is logical that the mechanics for A/B devices will not work
